funcs/func_cut.c

```diff
@@ -72,10 +72,14 @@
 				<para>Number of the field you want (1-based offset), may also be specified as a range (with <literal>-</literal>)
 				or group of ranges and fields (with <literal>&amp;</literal>)</para>
 			</parameter>
+ 					<example title="The varname parameter must be a variable name, not a string value. This is unusual syntax. So:">
```

Comment:
"Unusual" seems kind of normative to me... there are several dialplan functions that work similarly. Also example titles should be relatively short. Also it's a bit confusing as currently worded... variable names are string values. I think it'd be fine to rephrase to "The varname parameter must be a variable name, not a variable expression". A colon is not needed at the end.

Reply:
fix you wording ty ,
as for the usual in the dialpan most of the function use `${} `this one is different
`Set(foo=${CUT(bar,,${counter})})`
bar is whithout and counter would need to be with the `${} `